torch/lib/build_all.sh

```diff
@@ -36,14 +36,20 @@ else
     LDFLAGS="$LDFLAGS -Wl,-rpath,\$ORIGIN"
 fi
 C_FLAGS="$BASIC_C_FLAGS $LDFLAGS"
+NVCC_TARGETS="THC THCS THCUNN"
 function build() {
   mkdir -p build/$1
   cd build/$1
+  BUILD_C_FLAGS=''
+  eval 'case '$1' in
+      '${NVCC_TARGETS//" "/" | "}') BUILD_C_FLAGS=$C_FLAGS;;
```

Comment:
Have you verified that it still fixes it on PPC? That flag might be necessary when compiling `.c` files in THC.